### ISSUE TITLE
Imgurp seems to be shutdown.

### ISSUE THREAD

**Issue by SemihGuner** [seyahdoo/TabletopTurkeyifier#3]
Hey, Imgurp seems to be shutdown and doesn't work anymore. Since Imgur itself is unblocked currently, I'd suggest changing it back.

**Comment by seyahdoo** [comment]
Yeah, imgur seems to be back unbanned. will clear imgur from proxy list.
pastebinp website seems to be working on Turkcell, maybe the particular image the mod is trying to download is deleted from imgur.

**Comment by seyahdoo** [comment]
try again with the latest verson. it should be only proxying pastebin now.
